server/channels/store/sqlstore/group_store.go

```diff
@@ -261,7 +271,20 @@ func (s *SqlGroupStore) GetByIDs(groupIDs []string) ([]*model.Group, error) {
 
 func (s *SqlGroupStore) GetByRemoteID(remoteID string, groupSource model.GroupSource) (*model.Group, error) {
 	var group model.Group
-	if err := s.GetReplicaX().Get(&group, "SELECT * from UserGroups WHERE RemoteId = ? AND Source = ?", remoteID, groupSource); err != nil {
+	query, args, err := s.getQueryBuilder().
+		Select("*").
+		From("UserGroups").
+		Where(sq.Eq{
+			"RemoteId": remoteID,
+			"Source":   groupSource,
+		}).
+		ToSql()
+
+	if err != nil {
+		return nil, err
+	}
+
+	if err := s.GetReplicaX().Get(&group, query, args...); err != nil {
```

Comment:
You can also simplify the whole thing as:

```
s.GetReplicaX().GetBuilder(&group, builder) 
```

and pass just the builder. This method will automatically convert to query and args and also return error if necessary. This gets rid of the repetetive boilerplate.

Reply:
Thanks for the recommendation! I like the boilerplate reduction 😄